edm.yaml

```diff
@@ -192,21 +192,41 @@ datatypes :
      - fcc::CaloHit hits // Hits used in the cluster
 
   fcc::TrackState:
-    Description: "Track state at a given point on the track."
+    Description: "Track state at a given point on the track. The track parameters are chosen 
```

Comment:
Just to be sure, you propose to use these variables as TrackState members, and leave the description as it is?  if yes fine with me.